lib/easypost/services/billing.rb

```diff
@@ -59,7 +61,7 @@ def retrieve_payment_methods
     response = @client.make_request(:get, '/v2/payment_methods')
 
     if response['id'].nil?
-      raise EasyPost::Error.new('Billing has not been setup for this user. Please add a payment method.')
+      raise EasyPost::Exceptions::InvalidObjectError.new(EasyPost::Constants::NO_PAYMENT_METHODS)
```

Comment:
issue: This should raise a `PaymentError` per other libs

Reply:
Meant to talk to you about this. I saw that's what you did in Python, but it's this way in .NET

My argument: PaymentError is a status code-triggered exception, while this is more of a local error?

Reply:
Fair, I'll fix in Python. Looks like I did invalid object error in Node. PHP is PaymentError.